schemas/6.1/GitVersion.configuration.json

```diff
@@ -189,6 +189,13 @@
             "null"
           ]
         },
+        "paths": {
```

Comment:
please revert this file, please rebase on latest main and if needed you can regenerate this file